<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchedFilter</class>
 <widget class="QWidget" name="MatchedFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="ColorSelectionLayout">
     <item>
      <widget class="QPushButton" name="colorButton">
       <property name="text">
        <string> Pick Color</string>
       </property>
       <property name="icon">
        <iconset resource="MatchedFilter.qrc">
         <normaloff>:/icons/color.png</normaloff>:/icons/color.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="colorSample">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="SilderLayout">
       <item>
        <widget class="QLabel" name="thresholdLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Threshold:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="thresholdSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="thresholdValueLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>.3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MatchedFilter.qrc"/>
 </resources>
 <connections/>
</ui>
